README.md

```diff
@@ -49,3 +58,14 @@ Configure and build ns-3 from the `ns-3-dev` folder:
 ```
 
 This module does not provide Python bindings at the moment, they will be added in the near future.
+
+### Documentation ###
+
+To compile the documentation, please follow the instructions from the [ns-3 manual](https://www.nsnam.org/docs/manual/html/documentation.html).
+
+1. Istall the dcumentation-specific depdendencies as described in the [ns-3 installation guide](https://www.nsnam.org/wiki/Installation)
```

Comment:
Install

Is 1. correct in each bullet?

Reply:
Aye, it's part of the MD syntax.
Check [here](https://github.com/signetlabdei/qd-channel/blob/updating-app/README.md).